TourGuide_Horde/29_30_HillsbradFoothills.lua

```diff
@@ -1,8 +1,27 @@
 TourGuide:RegisterGuide("Hillsbrad Foothills (29-30)", "Arathi Highlands (30)", "Horde", function()
 return [[
+
+N Grind to 30... |LV|30| |C|Warrior|
+
 F Orgrimmar
 h Orgrimmar
 
+A The Islander |QUD|1718| |C|Warrior| |LV|30| |Z|Orgimmar| |N|(80.39, 32.38)|
```

Comment:
Since Line 4 already has |LV|30|, we can assume the player has reached the level required for these. Thus, having the tag isn't necessary, but technically doesn't hurt.

Reply:
Oh, and watch for "QID" spelling.